cachi2/core/config.py

```diff
@@ -0,0 +1,24 @@
+config = None
+
+class Config:
+    cachito_athens_url = None
+    cachito_bundles_dir = "./output"
+    cachito_sources_dir = "./sources"
```

Comment:
Removing unnecessary config attributes is planned for the future, right?

Reply:
I went ahead and removed them right away.